tensorflow/lite/experimental/swift/BUILD.apple

```diff
@@ -25,18 +25,47 @@ package(
 
 swift_library(
     name = "TensorFlowLite",
-    srcs = glob(["Sources/*.swift"]),
-    linkopts = [
-        "-Wl,-weak_framework,CoreML",
-        "-Wl,-weak_framework,Metal",
-    ],
+    srcs = glob(
+        [
+            "Sources/*.swift",
+        ],
+        exclude = [
```

Comment:
Would it be possible to do the selects here? I don't believe negating a select is possible, so if `use_coreml_delegate` is not true, then exclude `Sources/CoreMLDelegate.swift` as the "default" condition?

Same for Metal?

Reply:
I tried something like that (if I understand what you mean) but the value of `exclude` has to be a list of strings, it can't be a select, which was a TIL

Reply:
I see, was thinking something like:
```
    srcs = glob(
        [
            "Sources/*.swift",
        ],
        exclude = select({
          "//tensorflow/lite/tools/delegates:use_coreml_delegate": [],
          "//conditions:default": [
              "Sources/CoreMLDelegate.swift",
          ],
        }) + select({
          "//tensorflow/lite/tools/delegates:use_metal_delegate": [],
          "//conditions:default": [
              "Sources/MetalDelegate.swift",
          ],
        }),
    )
```

OR

```
    srcs = glob(
        [
            "Sources/*.swift",
        ],
        exclude = [] + select({
          "//tensorflow/lite/tools/delegates:use_coreml_delegate": [],
          "//conditions:default": [
              "Sources/CoreMLDelegate.swift",
          ],
        }) + select({
          "//tensorflow/lite/tools/delegates:use_metal_delegate": [],
          "//conditions:default": [
              "Sources/MetalDelegate.swift",
          ],
        }),
    )
```

But if that's less readable and still doesn't work, then I think what you have is reasonable. Was just looking for a way to avoid having to update files / selects in multiple places when adding new delegates in future.

Reply:
Yea we can't put the `select` in the `exclude` so this is what we're stuck with 😢